.github/workflows/gosec.yml

```diff
@@ -0,0 +1,28 @@
+name: gosec
+
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '0 5 * * *' # Daily at 05:00 UTC
+
+jobs:
+  coverity:
+    name: "🔎 gosec check"
+    if: github.repository == 'osbuild/image-builder'
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on
+    steps:
+
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Run Gosec Security Scanner
+        uses: securego/gosec@master
```

Comment:
I'd rather pin this to a version and update it separately. This way, we don't see unrelated PRs failing when `gosec` changes or becomes more strict.